src/Menu/Menu.js

```diff
@@ -48,23 +51,26 @@ export const MenuItem = ({ url, link, isLink, separator, addon, children, onclic
 
 MenuItem.propTypes = {
     addon: PropTypes.string,
+    addonProps: PropTypes.object,
     className: PropTypes.string,
     isLink: PropTypes.bool,
+    linkProps: PropTypes.object,
     separator: PropTypes.bool,
     url: PropTypes.string
 };
 
 // ---------------------------------------- Menu Group ----------------------------------------
-export const MenuGroup = ({ title, children, className, ...props }) => {
+export const MenuGroup = ({ title, children, className, titleProps, ...props }) => {
     return (
         <div className={`fd-menu__group${className ? ' ' + className : ''}`} {...props}>
```

Comment:
Although not changed, this should probably move the spread to be first.

Reply:
Done